docs/Developers/Newcomers_Guide.rst

```diff
@@ -336,9 +336,16 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
```

Comment:
commit message: "Change in note" -> a commit obviously changes something ;)
You should be more explicit, try `Newcomers_Guide.rst: Emphasize on low issues` :)